en-US/firefox/adblocker.lang

```diff
@@ -0,0 +1,131 @@
+One of the most popular ad blockers for Chrome, Safari and Firefox is <a href="%(url)s">AdBlock</a>. Use it to block ads on Facebook, YouTube and Hulu.
+
+
+;Create a tracker-free zone with Content Blocking
+Create a tracker-free zone with Content Blocking
+
+
+;On Firefox, you can use <a href="%(privacy)s">Privacy</a> or <a href="%(blocking)s">Content Blocking</a> settings to get even more control over ad trackers that serve you the ads.
+On Firefox, you can use <a href="%(privacy)s">Privacy</a> or <a href="%(blocking)s">Content Blocking</a> settings to get even more control over ad trackers that serve you the ads.
+
+
+;Choose your level of protection
+Choose your level of protection
+
+
+;To start, click on the Firefox menu in the top right-hand corner of your screen. It looks like three lines stacked on top of each other. In the drop-down menu, click on Content Blocking. You should see a blue pop-up with different selections.
+To start, click on the Firefox menu in the top right-hand corner of your screen. It looks like three lines stacked on top of each other. In the drop-down menu, click on Content Blocking. You should see a blue pop-up with different selections.
+
+
+;Go easy with Standard mode
+Go easy with Standard mode
+
+
+;If ads don’t bother you and you don’t mind being followed by trackers and third-party cookies, then the Standard setting should work for you. To get trackers off your tail in Standard mode, use a <a href="%(url)s">Private Browsing</a> window.
+If ads don’t bother you and you don’t mind being followed by trackers and third-party cookies, then the Standard setting should work for you. To get trackers off your tail in Standard mode, use a <a href="%(url)s">Private Browsing</a> window.
+
+
+;Get tough with Strict mode
+Get tough with Strict mode
+
+
+;If seeing too many ads ruins your day, then the Strict mode is a better fit. This mode will block known third-party trackers and cookies in all Firefox windows.
+If seeing too many ads ruins your day, then the Strict mode is a better fit. This mode will block known third-party trackers and cookies in all Firefox windows.
+
+
+;Do-it-yourself Custom mode
+Do-it-yourself Custom mode
+
+
+;The custom setting gives you the ultimate choice. You can decide what you’re blocking, including trackers, cookies and more. If you allow cookies from a website, you’ll automatically be in Custom mode.
+The custom setting gives you the ultimate choice. You can decide what you’re blocking, including trackers, cookies and more. If you allow cookies from a website, you’ll automatically be in Custom mode.
+
+
+;Cover your trail, block trackers
+Cover your trail, block trackers
+
+
+;Click on the trackers box and you’ll be able to block trackers in two ways. One way to block trackers is to do it when you’re working in a Private Window. Another way to do it is to block trackers in all windows. Keep in mind that if you choose to always block trackers, some pages might not load correctly.
```

Comment:
Shouldn't the `trackers box` be `Trackers box` (capitalized T) - it is a UI term this is referring to.

Reply:
While `trackers` is capitalized in the UI, it is not a proper noun or name so it should not be capitalized here.

Reply:
It is the feature  though. If you look how `Custom`, `Strict" mode mentions are capitalized in the strings, because they are referring to the features, which is what this one is for.

Reply:
Oh! I see what you mean now. Ok I will update. 😄

Reply:
Make sure to update the template too.

Reply:
I will have a PR for the template in bedrock once the changes here are finalized. Thanks so much for your help, by the way!